third_party/terraform/tests/resource_redis_instance_test.go.erb

```diff
@@ -69,6 +70,37 @@ func TestAccRedisInstance_regionFromLocation(t *testing.T) {
 	})
 }
 
+
+<% unless version == 'ga' -%>
+func TestAccRedisInstance_redisInstanceAuthEnabled(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"random_suffix": randString(t, 10),
+	}
+
+	vcrTest(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		ExternalProviders: map[string]resource.ExternalProvider{
+			"random": {},
+		},
+		CheckDestroy: testAccCheckRedisInstanceDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccRedisInstance_redisInstanceAuthEnabled(context),
```

Comment:
Do you mind adding a second update step, which sets `auth_enabled` to false? Generally tests that update to the zero value for a type tend to catch a lot of weird behaviour in APIs.

Reply:
done!